python_bindings/bindings.cpp

```diff
@@ -108,7 +106,7 @@ class Index {
     default_ef=10;
   }
 
-  static const int ser_version = 1; // serialization version
+  static const int ser_version = 2; // serialization version
```

Comment:
I think that should be still 2, since there are no changes to the format.

Reply:
Good catch, thank you!